cuegui/cuegui/Main.py

```diff
@@ -92,9 +92,14 @@ def startup(app_name, app_version, argv):
     cuegui.Style.init()
 
     # If the config file does not exist, copy over the default
+    if os.getenv('CUEGUI_DEFAULT_INI_PATH'):
+        default_ini_path = os.getenv('CUEGUI_DEFAULT_INI_PATH')
+    else:
+        default_ini_path = cuegui.Constants.DEFAULT_INI_PATH
```

Comment:
Hm, I'm wondering if it wouldn't be better to do this in `cuegui.Constants`. That way any potential future code that refers to `DEFAULT_INI_PATH` won't have to apply the same logic as is applied here.

`cuegui.Constants` doesn't at the moment do any env var checks, though it *does* apply some similar logic about the current environment e.g. `__file__`.

@gregdenton Any thoughts here?

Reply:
Also I believe this block could be rewritten like:

```
default_ini_path = os.getenv('CUEGUI_DEFAULT_INI_PATH', cuegui.Constants.DEFAULT_INI_PATH)
```

Reply:
I agree with having this logic in one place. `Constants.py` is a bit unfortunate name for this file, but given the current structure it does seem the best spot for this. So the proposed change is in  `Constants.py`:
`DEFAULT_INI_PATH = os.getenv('CUEGUI_DEFAULT_INI_PATH', os.path.dirname(__file__) + '/config')`